image_geometry/src/pinhole_camera_model.cpp

```diff
@@ -1,5 +1,5 @@
 #include "image_geometry/pinhole_camera_model.h"
-#include <sensor_msgs/distortion_models.h>
+#include <sensor_msgs/distortion_models.hpp>
```

Comment:
yeah it's just that anybody wanting to migrate a package using this will have to have this extra migration step. I don't think it's the end of the world either, just trying to see how many extra steps we can avoid end users to have to go through. Same logic made me ask you to match the ROS1 folder structure for all the sensor_msgs utilities.

I don't feel very strongly about one or the other I just wanted to know what you think about it